README.md

```diff
@@ -119,8 +119,9 @@ import { LifetimeWarrantyIcon } from '@assets/svg';
 
 When you need to update the Shopify storefront GraphQL schema version, follow these steps:
 
-1. Update `NEXT_PUBLIC_SHOPIFY_STOREFRONT_VERSION` in `frontend/.env.development` and `frontend/.env.production`
-2. Run `pnpm codegen:download-shopify-storefront-schema`
+1. Install `graphqurl`: `npm install -g graphqurl@1.0.1`
```

Comment:
This shouldn't be needed, as `graphqurl` is already a project dependency

Reply:
I hunted all over the codebase, then npm for a cli tool named `gq`. Took me a while to trace that to a package named `graphqurl`. Didn't realize it was already a dependency until you mentioned it.

Anyway, now that I realize this is a dependency, this docs change isn't needed.